modules/custom/az_demo/data/az_demo_image_file.json

```
@@ -188,6 +188,14 @@
       "caption" : "Insulix-21 is a novel therapy that combines cutting-edge stem cell technology with advanced immunomodulation techniques",
       "credit" : "Courtesy of Wilma Wildcat",
       "uuid": "aec26a34-72da-4a7c-94e1-8e2793ec67c2"
+    },
+    {
+      "filename" : "1386px-Leonardo_da_Vinci_-_presumed_self-portrait_-_WGA12798.jpg",
+      "alt" : "Self-portrait of Leonardo da Vinci",
+      "title": "Self-portrait of Leonardo da Vinci",
+      "caption": "Leonardo da Vinci was an Italian polymath of the Renaissance whose areas of interest included invention, painting, sculpting, architecture, science, music, mathematics, engineering, literature, anatomy, geology, astronomy, botany, writing, history, and cartography.",
+      "credit": "By Leonardo da Vinci - https://www.latitudinex.com/europa/leonardo-valle-della-loira-rinascimento.htmlWeb Gallery of Art:   Image  Info about artwork, Public Domain, https://commons.wikimedia.org/w/index.php?curid=15497207",
```

The first link in the credit is broken and the [new URL](https://www.latitudinex.com/europa/valle-della-loira-tra-rinascimento.html) doesn't seem to have the image anymore. I think we could just use Wikipedia as the source.

```suggestion
      "credit": "By Leonardo da Vinci. Image source: Wikipedia, https://commons.wikimedia.org/w/index.php?curid=15497207. Public domain.",
```